clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -103,14 +103,47 @@ private static void schemaToFieldTableHtml(Schema schema, StringBuilder b) {
         b.append("<th>Description</th>\n");
         b.append("</tr>");
         for (BoundField field : fields) {
-            b.append("<tr>\n");
-            b.append("<td>");
-            b.append(field.def.name);
-            b.append("</td>");
-            b.append("<td>");
-            b.append(field.def.docString);
-            b.append("</td>");
-            b.append("</tr>\n");
+            if (!(field.def.type instanceof TaggedFields)) {
```

Comment:
Hello @AndrewJSchofield, Thanks for your PR. 
Leave a small suggestion: using POSITIVE logic here should be better than using NEGATIVE logic.

Reply:
I did consider this but I chose to put the non-tagged fields case first.